merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/types/ValueSchema.java

```diff
@@ -516,15 +516,13 @@ public Boolean onVariant(final List<Variant> variants) {
     });
   }
 
-  public static final class Variant {
-    /** The unique internal name of this variant. */
-    public final String key;
-    /** The user-facing presentation of this variant. */
-    public final String label;
-
-    public Variant(final String key, final String label) {
-      this.key = Objects.requireNonNull(key);
-      this.label = Objects.requireNonNull(label);
+  public static record Variant(
+      String key,  // The unique internal name of this variant
+      String label // The user-facing presentation of this variant
+  ) {
+    public Variant {
+      Objects.requireNonNull(key);
+      Objects.requireNonNull(label);
```

Comment:
I'm much less bullish on keeping these `requireNonNull`s around these days, especially when the `record` could otherwise be a one-liner. They certainly don't hurt though 😅